apiextensions/storageversion/migrator.go

```diff
@@ -38,6 +40,10 @@ type Migrator struct {
 	apixClient    apixclient.Interface
 }
 
+type MigrateConfig struct {
+	NoFailIfCRDFound bool `envconfig:"MIGRATE_CRD_IF_FOUND_NO_FAILURE" default:"false"`
+}
+
```

Comment:
It's not obvious that consuming this API will read env-vars so let's add a `IgnoreNotFound` bool to the `Migrator` struct and do the lookup in the `main` of the cmd at https://github.com/knative/pkg/blob/45c37c266b74eed0c2b32f537220d5da7cdfe077/apiextensions/storageversion/cmd/migrate/main.go#L34

Reply:
@dprotaso ok to change the signature of [NewMigrator](https://github.com/knative/pkg/pull/2493/files#diff-9a19a801068606c877c96a2e53cac2bc639abcd220540f160aa90b582c0a6471R44) to add this new bool ?